index.html

```diff
@@ -181,6 +181,66 @@ <h2>Requests/responses sharing and proxy between Native and WebView</h2>
         </dl>
       </section>
 	    
+      <section>
+        <h2>The Origin in a WebView for locally hosted content</h2>
+        <dl>
+            <dt>Submitter(s)</dt>
+            <dd>
+            Niklas Merz, Apache Software Foundation
+            </dd>
+            
+            <dt>Motivation</dt>
+            <dd>
+            <a href="https://appfigures.com/top-sdks/development/all">Up to 30%</a> of apps found in the App stores (Apple and Google) are built with frameworks like <a href="https://cordova.apache.org">Apache Cordova</a> and <a href="https://capacitorjs.com/">CapacitorJS</a>. Those two frameworks use one big Webview for providing app developers a native wrapper and some plugins for their Web App. App developers build their Web application and put the HTML, CSS and JavaScript files in one folder. The framework then takes care of building a Native App project and bundling the Web code as a Native application ready to distribute via the App stores.
```

Comment:
```suggestion
           WebViews are widely used for building apps on the dominating mobile and desktop platforms. <a href="https://appfigures.com/top-sdks/development/all">Up to 30%</a> of apps found in the app stores (Apple and Google) are built with frameworks like <a href="https://cordova.apache.org">Apache Cordova</a> and <a href="https://capacitorjs.com/">CapacitorJS</a>. Those two frameworks use one big WebView for providing app developers a native wrapper and some plugins for their Web app. App developers build their Web application and put the HTML, CSS and JavaScript files in one folder. The framework then takes care of building a native app project and bundling the Web code as a native application ready to distribute via the app stores.
```

Small intro and some capitalization fixes. Excuse my spelling. Switching from German to English sometimes makes me mix up the capitalization.